runtime/compiler/env/VMJ9Server.cpp

```diff
@@ -1515,6 +1508,26 @@ TR_J9ServerVM::getROMMethodFromRAMMethod(J9Method *ramMethod)
    return std::get<0>(stream->read<J9ROMMethod *>());
    }
 
+TR_OpaqueClassBlock *
+TR_J9ServerVM::getCacheClassFromSignature(const char * sig, int32_t sigLength, bool isVettedForAOT, TR_OpaqueMethodBlock *methodBlock, J9ClassLoader * cl, bool cacheInvalidatedClass)
```

Comment:
The name of the method should better reflect what the method does. There is another cache for classes, so there could be some confusion.